doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -30,10 +56,51 @@
 > * Which optional features are used and which optional API-s are available
 > * Which [alfa or beta features](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) are used
 
+The distribution and version of Kubernetes used must be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) which lists all those that are Certified as being conformant with the CNCF Kubernetes specification.
+
+This Reference Architecture also specifies:
+
+- Master nodes must run the following Kubernetes control plane services:
+    - kube-apiserver
+    - kube-scheduler
+    - kube-controller-manager
+- Master nodes can also run the etcd service and host the etcd database, however etcd can also be hosted on separate nodes
+- In order to support `req.gen.rsl.01`, `req.gen.rsl.02` and `req.gen.avl.01` a Reference Implementation must:
+    - Consist of either three, five or seven nodes running the etcd service (can be colocated on the master nodes, or can run on separate nodes)
```

Comment:
How about co-locating in the worker nodes?

Reply:
Thanks, fixed.